<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormPresupuestoBase</class>
 <widget class="QWidget" name="FormPresupuestoBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>547</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Datos del presupuesto</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="LNumeroComprobante">
          <property name="text">
           <string>Numero:</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::MinimumExpanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="_3">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Fecha:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateEdit" name="dEFecha">
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Cliente</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="ECBClientes" name="CBCliente">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Direcci√≥n:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="LEDireccion"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="GBContenido">
     <property name="title">
      <string>Contenido</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="EDSBPrecio" name="DSBCant">
          <property name="maximum">
           <double>1000000000.000000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="ECBProductos" name="CBProductos">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="PBAgregar">
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableView" name="TVContenido"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="PBAgregarDescuento">
          <property name="text">
           <string>Descuento</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="PBEliminarDescuento">
          <property name="text">
           <string>Descuento</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="PBEliminar">
          <property name="text">
           <string>Eliminar</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="PBEliminarTodo">
          <property name="text">
           <string>Borrar Todo</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="PTEObservaciones">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="plainText">
      <string>Observaciones: </string>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::TextEditorInteraction</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ECBProductos</class>
   <extends>QComboBox</extends>
   <header>ecbproductos.h</header>
  </customwidget>
  <customwidget>
   <class>EDSBPrecio</class>
   <extends>QDoubleSpinBox</extends>
   <header>edsbprecio.h</header>
  </customwidget>
  <customwidget>
   <class>ECBClientes</class>
   <extends>QComboBox</extends>
   <header>ecbclientes.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
